epsagon/utils.py

```diff
@@ -98,6 +98,7 @@ def init(
     propagate_lambda_id=False,
     logging_tracing_enabled=True,
     step_dict_output_path=None,
+    sample_rate=1,
```

Comment:
yet another default :) 
Please create a const DEFAULT_SAMPLE_RATE and use it instead of hard code it in different places.